lib/console-log.coffee

```diff
@@ -65,7 +65,9 @@ module.exports =
         editorLineCount = editor.getLastScreenRow()
         functionCheckValues = ['=>', 'function', '){', ') {']
         identifierCaseConfig = configProp 'console-log.identifierCase'
-        identifier = (
+        identifierPrefixConfig = configProp 'console-log.identifierPrefix'
+        identifierPrefix = (identifierPrefixConfig || '').replace /('|\\)/g, '\\$1'
+        identifier = identifierPrefix + (
```

Comment:
You can `replace` only once. And make the whole thing a bit cleaner while you're at it :)
```coffee
casedSelectedText = if identifierCaseConfig
  then selectedText else selectedText.toUpperCase()
identifierPrefix = identifierPrefixConfig || ''
identifier = (identifierPrefix + casedSelectedText).replace /('|\\)/g, '\\$1'
```

Reply:
that sounds right